server/jetstream_test.go

```diff
@@ -7417,17 +7417,15 @@ func TestJetStreamRequestAPI(t *testing.T) {
 		t.Fatalf("Created time seems wrong: %v\n", scResp.Created)
 	}
 
-	checkBadRequest := func(e *ApiError, description string) {
+	checkNatsError := func(e *ApiError, id APIErrorIdentifier) {
```

Comment:
do we have a more generic place for test helper functions? I suspec this will be used quite a lot